fiftyone/utils/cvat.py

```diff
@@ -3498,6 +3500,7 @@ def __init__(self, name, url, username=None, password=None, headers=None):
         self._password = password
         self._headers = headers
 
+        self._orginization = None
```

Comment:
This organization parameter is not yet being set anywhere, I would pass it in through here when the CVATAnnotationAPI object is being created: https://github.com/voxel51/fiftyone/blob/c47ce83e8400b16b58ff8787e868aac5db06b267/fiftyone/utils/cvat.py#L3232

In that case, it would also be good to add getters and setters similar to the other properties of the config: https://github.com/voxel51/fiftyone/blob/c47ce83e8400b16b58ff8787e868aac5db06b267/fiftyone/utils/cvat.py#L3143